tsconfig.json

```diff
@@ -14,7 +14,8 @@
     "resolveJsonModule": true,
     "isolatedModules": true,
     "noEmit": true,
-    "jsx": "react-jsx"
+    "jsx": "react-jsx",
+    "declaration": true
```

Comment:
I had to add this declaration property for the sample application - it was complaining it couldn't get types. Not sure on the wider implications for release.

Reply:
Removed now @knagurski